### PR TITLE
chore(helm): add jobteaser repo

### PR DESCRIPTION
**Pull request by lerrigatto**

### CODE REVIEW

File: orbs/helm/orb.yml [hunk 1]
```diff
@@ -149,6 +149,7 @@ jobs:
             helm init --client-only
             helm plugin install https://github.com/chartmuseum/helm-push
             helm repo add --username <<parameters.helm_repo_user>> --password <<parameters.helm_repo_pass>> jt <<parameters.helm_repo_url>>
+            helm repo add jobteaser https://jobteaser.github.io/charts
```
Comment (galdorjt) [inline orbs/helm/orb.yml]: I do not get it, why is there a `jt` repository *and* a `jobteaser` repository ?

Comment (lerrigatto) [reply orbs/helm/orb.yml]: The first one is the private chartmuseum one, the second one is the public github one.
We could/should move the github one to chartmuseum...

Comment (lerrigatto) [reply orbs/helm/orb.yml]: Maybe we can have better naming, I don't think helm will care about how you call the repos.

Comment (galdorjt) [reply orbs/helm/orb.yml]: Helm won't care; the people we will try to understand what is going on will. Why not name them `jobteaser-private` and `jobteaser-public` ? It makes it obvious and, the information add an extra layer of safety.

**Pull request merged**